.github/workflows/release-homebrew.yaml

```diff
@@ -85,8 +88,34 @@ jobs:
           brew bump-cask-pr --write-only -f --version=$FINCH_VERSION finch
       - name: Silently install
         run: |
+          export HOMEBREW_NO_INSTALL_FROM_API=1
           cd $(brew --repo homebrew/cask)
           brew install --cask ./Casks/finch.rb
+      - name: Build project
+        run: |
+          brew install go lz4 automake autoconf libtool
+          export PATH="/opt/homebrew/opt/libtool/libexec/gnubin:$PATH"
+          which libtool
+          make
+      - name: Multiple instances of Finch test
+        run: |
+          pwd
+          # start two Finch VM instances
+          ./_output/bin/finch vm init
+          finch vm init
+          # start a container in each VM instance
+          ./_output/bin/finch pull alpine
+          finch pull alpine
+          ./_output/bin/finch run -d --name test-ctr1 alpine
+          finch run -d --name test-ctr2 alpine
+          ./_output/bin/finch vm stop && ./_output/bin/finch vm remove
+          finch vm stop && finch vm remove
+      - name: Clean up multiple instance test
+        run: |
+          sudo rm -rf ./_output
+          export HOMEBREW_NO_INSTALL_FROM_API=1
+          cd $(brew --repo homebrew/cask)
+          brew reinstall --cask ./Casks/finch.rb
```

Comment:
Need to reinstall after the multiple instance test because there are errors when init the VM in e2e tests. https://github.com/runfinch/finch/actions/runs/4367457552/jobs/7638794529 I found some users had similar issue as this and resolved by reinstalling.

Reply:
Should we add this info as code comment?

Reply:
Added to comment. Thanks!